AUTHORS

```diff
@@ -475,3 +475,4 @@ Jason Wohlgemuth <jhwohlgemuth@users.noreply.github.com>
 Ryan Graham <r.m.graham@gmail.com>
 Hirse <jan.pilzer@gmx.de>
 Colin Rotherham <work@colinr.com>
+Kin Lum <kenneth.kin.lum@gmail.com>
```

Comment:
Please don't patch AUTHORs directly. We generate it from the commit history on every publish.